funnel/models/event.py

```diff
@@ -28,7 +28,30 @@ def make_private_key():
     )
 
 
-class Event(BaseScopedNameMixin, db.Model):
+class ScopedNameTitleMixin(object):
```

Comment:
Derive this from `BaseScopedNameMixin` so you don't need the class again in the use cases below. It's easy to get confused about method resolution order in base classes and order them the wrong way (L-R is correct), but hard to get confused about parent-child relationships. Since this class replaces methods in `BaseScopedNameMixin`, derive from it.